gitops/argocd/argo-apps/tekton-results.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
```

Comment:
For the customers to be able to use Tekton-results, we'll need to update `/images/cluster-setup/install.sh` as well.

Reply:
As discussed, this will be completed with PLNSRVCE-422 but it's dependent on PLNSRVCE-421 being completed first.